modules/Bio/EnsEMBL/BaseAlignFeature.pm

```diff
@@ -270,6 +270,8 @@ sub alignment_length {
 
   if ($self->{'align_type'} eq 'ensembl') {
     return $self->_ensembl_cigar_alignment_length();
+  } elsif ($self->{'align_type'} eq 'mdtag') {
```

Comment:
We have a problem with a lack of comments in our code. As we go through updating things can we take the opportunity to put some inline comments? For example here just briefly explaining what those alignment types are.